### PR TITLE
add Mamba MK4 F405Mini ver a and b

### PR DESCRIPTION
**Pull request by Hwurzburg**
note: #23245 corrects defaults

### CODE REVIEW

File: common/source/docs/common-mambaf405-mini.rst [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+-  **Processor**
+
+   -  STM32F405 32-bit processor running at 168 MHz
+   -  OSD - AT7456E
+   -  128Kb (16MB) Data Log Flash (:ref:`LOG_BACKEND_TYPE<LOG_BACKEND_TYPE>` = 2)
+
+-  **Sensors**
+
+   -  Dual MPU6000 (Version A) or IMC42688 (Version B)
```
Comment (andyp1per) [inline common/source/docs/common-mambaf405-mini.rst:25]: I don't think they are dual are they?

File: common/source/docs/common-mambaf405-mini.rst [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+Firmware
+========
+
+Firmware for this board can be found `here <https://firmware.ardupilot.org>`_ in  sub-folders labeled "Mamba MK4 F405Mini"".
+
+Loading Firmware
+================
+
+Firmware for this board can be found `here <https://firmware.ardupilot.org>`_ in  sub-folders labeled "KakuteH7-Wing".
```
Comment (andyp1per) [inline common/source/docs/common-mambaf405-mini.rst:153]: typo

File: common/source/docs/common-mambaf405-mini.rst [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+RSSI
+====
+
+Analog RSSI input, if used, :ref:`RSSI_ANA_PIN<RSSI_ANA_PIN>` should be set to "12".
+
+Firmware
+========
+
+Firmware for this board can be found `here <https://firmware.ardupilot.org>`_ in  sub-folders labeled "Mamba MK4 F405Mini"".
```
Comment (andyp1per) [inline common/source/docs/common-mambaf405-mini.rst:148]: Directory doesn't exist - think you mean MambaF405-2022

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: common/source/docs/common-mambaf405-mini.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-mambaf405-mini.rst:25]: right....badly worded....

File: common/source/docs/common-mambaf405-mini.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-mambaf405-mini.rst:153]: missed is a global replace...will fix

File: common/source/docs/common-mambaf405-mini.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-mambaf405-mini.rst:148]: again a global replace messup...will fix...thanks

**Pull request merged**